index.d.ts

```diff
@@ -696,7 +696,7 @@ interface SessionConfigParams {
    * cryptographically strong random value of sufficient size or sign the cookie
    * by setting {@Link signSessionStoreCookie} to `true`.
    */
-  genid?: (req: OpenidRequest) => string;
+  genid?: (req: OpenidRequest) => Promise<string>;
```

Comment:
```suggestion
  genid?: (req: OpenidRequest) => Promise<string> | string;
```

so you can also use sync

Reply:
yep indeed, thanks for the change!